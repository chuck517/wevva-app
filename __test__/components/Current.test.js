import React from 'react';
import renderer from 'react-test-renderer';
import Current from '../../components/Current';


it('renders without crashing', () => {
  const rendered = renderer.create(<Current />).toJSON();
  expect(rendered).toBeTruthy();
});

it('test against snapshot', () => {
  const tree = renderer.create(<Current />).toJSON();
  expect(tree).toMatchSnapshot();
});