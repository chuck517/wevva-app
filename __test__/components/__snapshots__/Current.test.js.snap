// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Current /> renders correctly across screens 1`] = `
<ActivityIndicator
  animating={true}
  color="#fff"
  hidesWhenStopped={true}
  size="large"
/>
`;

exports[`test against snapshot 1`] = `
<ActivityIndicator
  animating={true}
  color="#fff"
  hidesWhenStopped={true}
  size="large"
/>
`;
